libs/led-matrix/include/LEDMatrix.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include "hw_bridge.h"
+#include "mbed.h"
+
+class LEDMatrix {
+ public:
+  // Define LED matrix colour channels by the pins it is connected to.
+  LEDMatrix(PinName R, PinName G, PinName B);
+
+  // Terminate thread and clear lights.
+  ~LEDMatrix();
+
+  // Set the state of the LEDs.
+  void setState(HWBRIDGE::LEDMATRIX::LEDMatrixState state);
+
+  // Periodically flash the colour given by R, G, B on the LED matrix.
+  void setFlashColor(bool R, bool G, bool B);
+
+  // Set a solid colour given by the R, G, B on the LED matrix.
+  void setSolidColor(bool R, bool G, bool B);
+
+ private:
+  DigitalOut m_RChannel;
+  DigitalOut m_GChannel;
+  DigitalOut m_BChannel;
+
+  // Flags
+  constexpr static uint32_t START_FLASH = 1UL;  // 01
+  constexpr static uint32_t ENDED_FLASH = 2UL;  // 10
```

Comment:
```suggestion
  constexpr static uint32_t START_FLASH   = (1U << 0);
  constexpr static uint32_t ENDED_FLASH = (1U << 1);
```
This is more expressive of intent